app/src/main/java/com/infomaniak/mail/utils/FetchMessagesManager.kt

```diff
@@ -213,21 +214,21 @@ class FetchMessagesManager @Inject constructor(
         if (message.isSeen) return true
 
         val formattedPreview = message.getFormattedPreview(appContext)
-        val body = if (message.body?.value.isNullOrBlank()) {
-            formattedPreview
-        } else {
+        val notificationBody = "\n" + if (formattedPreview is FormatedPreview.Body) {
             message.body
                 ?.let {
                     val content = MessageBodyUtils.splitContentAndQuote(it).content
                     val dirtyDocument = content.removeLineBreaksFromHtml()
                     val cleanedDocument = HtmlSanitizer.getInstance().sanitize(dirtyDocument)
-                    return@let "\n${cleanedDocument.wholeText().trim()}"
+                    return@let cleanedDocument.wholeText().trim()
```

Comment:
Why do you remove the `\n`

Reply:
Because I move it here. We add `\n` in every case
```kotlin
val notificationBody = "\n" + if (formattedPreview is FormatedPreview.Body) {
```